libs/portals/admin/ids-admin/src/components/forms/EditApplication/EditApplication.action.ts

```diff
@@ -220,7 +223,12 @@ export const schema = {
         }),
     })
     .merge(defaultSchema),
-  [ClientFormTypes.permissions]: defaultSchema,
+  [ClientFormTypes.permissions]: z
+    .object({
+      addedScopes: z.string().transform(splitStringOnCommaOrSpaceOrNewLine),
+      removedScopes: z.string().transform(splitStringOnCommaOrSpaceOrNewLine),
```

Comment:
After saving the settings we need to reset these arrays to empty?

Reply:
are you saying that we need to clear these values in the Zod object, or on the context ?